### PR TITLE
Correcciones en Salud y Seguridad

### PR DESCRIPTION
**Pull request by diegobeauroyre**

### CODE REVIEW

**Comment by diegobeauroyre** [comment]
Cómo lo ven @rodowi @fer-gandara

File: _posts/historias/salud/2015-01-11-mortalidad-materna-recomendaciones.md [hunk 1]
```diff
@@ -81,16 +81,16 @@ Desarrollamos una tubería para nuestro análisis en el que se ejecutan todas la
...
 
-Para los puntos 2 y 3, encontramos que, dada la misma población, las madres que dan a luz por cesárea en los hospitales de la Secretaría de Salud tienen una tasa de mortalidad mucho más alta que los que están en los hospitales del Sector Salud. Además, las madres del Seguro Popular que asisten los hospitales del Sector Salud (y por lo tanto tienen un desfase de seguros) tienen mucho más altas tasas de mortalidad que aquellos que sin son beneficiarios, dada la misma población. 
+Para los puntos 2 y 3, encontramos que, dada la misma población, las madres que dan a luz por cesárea en los hospitales de la Secretaría de Salud tienen una razón de mortalidad mucho más alta que los que están en los hospitales del Sector Salud. Además, las madres del Seguro Popular que asisten los hospitales del Sector Salud (y por lo tanto tienen un desfase de seguros) tienen mucho más alta razón de mortalidad que aquellos que sí son beneficiarios, dada la misma población.
```
Comment (rodowi) [inline _posts/historias/salud/2015-01-11-mortalidad-materna-recomendaciones.md]: "que si son beneficiarios" no es afirmación es condicional

**Comment by rodowi** [comment]
:+1:

**Comment by diegobeauroyre** [comment]
@rodowi @fer-gandara To merge or not to merge?

![Alt text](data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBxQTEhUUExQVFhQXFxoXFxcXGBwXHBwXHBwcHBcaFxcYHCggGBwlHBgcITEhJSkrLi4uGB8zODMsNygtLisBCgoKBQUFDgUFDisZExkrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrKysrK//AABEIALoBDwMBIgACEQEDEQH/xAAcAAACAwEBAQEAAAAAAAAAAAAEBQIDBgcBAAj/xAA8EAABAwIEBAQDBgUDBQEAAAABAAIRAyEEBRIxQVFhcQYTIoGRobEjMlLB0fAHFELh8TNiciSCssLSFv/EABQBAQAAAAAAAAAAAAAAAAAAAAD/xAAUEQEAAAAAAAAAAAAAAAAAAAAA/9oADAMBAAIRAxEAPwDROUCVN4USgrhfKS8hBAuXhcpFqjoQRJVFXENbuUDmmbtYfLYC+q6QGt5xxWap4p+pzSdeg/acr7x2AQa1uYgyAPUOBtbpwUKWYhxiw6E3+CyWFHm1Hta7y61Mk07wHs4RyPS8T8HGHxoJ01QGVhuHgtnhMi3vsgdirPEdlcwE8R9EppVg07FsbwNf/iJCb4FzaolpB4WMhBJ+HcP6T7IV5PHfkmtLDOYZExy/tw+SZ0RTqjRVaO+zm9j+RQZRehHZjljqTubeDuffkeiFIEII0xxUnqTRZQc66CIavCFMBfOCCNNtiqwLojDt3UmU7ElBTXaAIG6+wmHmVEooelndAuxbRMBDOZBCKcxSxlDTplAuqC5Q7wisUhXuQUuVlIWKrcrNmIBiVWQXFWBmoq2AEH1IRZEUmXVLbo2jTCDXPF1HSr3sXoagoNNeeWiQ1Qe1AMWofEVg3qTYAGLm26JruDQSeSRk+rzH2A2B3nmgsy3II14h9js2eXSdpO55Duk1DACnXcwEv1ua5zosQdLC0Hl6j8RyTw459X0AwPkBtJ68Y4QjcD4acakNu1oYG9ACH3P/ACF/jxQYbMMv00w8btJAO1mudNzzaB8EVgcYysG0q/qE+ip/Ux3CTw5TcHitt4qypjKQptu0NIJ4lxA0/WT3lczwDC2qwcNYBG3HtccEGgwlMiqaVcw9p0tdcGDdpa7cW4Gy0NHAkPAqSDHprNsT0cRY9nCO6I8ceHhUDnNkO0xIngJEx3+KS5Pn1VjhhsQJdOlj2kt1bWJ57b79Cg3OBoTZ+42cBHxH6SEYzCtBAJA5Tx/48fZI6GYkD1Oa0X03JmLkRN3De4um2T5g57R9oyqCbWA6jbYxtz5oG7MNTLeDmn3BWS8SZQKZDqX3TuOIPSd0/flIa7zaRLXnc8xyqC2uOe46oHG5k17vKc31X22Mb9uxg3CDJ6bL2nSRNalDiBtJXrWoICmvnU1c2F6QgqwlL1L7Fi8DZe0z6oUnMQBaFLEusFPXYodry4II0myR3CJz5mks9lTQEOHdW+JjJaQgUY+8Ql7mFGViotQCNpFe4ykQAE1yqlrqAcFf4qY1j2wECXA4WWlTbh0xwEERzCgKV0FDKK81w6AjWsVuGwl5QaioxVkIyoxQLUAs9FXVJAKOaxD5m37N8fhP0QZdua6nTplnOfmrH1wdiCORj5FLsp0vbB3A47Ee1wjDlZBmi4AxOh0/KN0F2ExlNm1MuP8AsaXX6wtBTzwhlgRaYiI7jdZV+HqiZdpPJogd5IFlLDMgE6hxkjaeN+J4fFA2zDG69xsJI59J5nYnlbisrkuCaardXq0nXYwLbyeQP16p1TwoqAFxPbae/FOcpwFIDToCBhg8aaxkt6DlYEfRE4nIqNaNbQXDjyPMHhursPTDRYAAWR2GaCgFp+GqLrObN54i44249d0M7wtoY8UTpmwBJgidXsd/2VpaDIV7igzzsLWexkmHt3JO/Oef9lLF5YwDVHqAN+n9oHwTlwVNZlkGExA9RVcIrM2xUcORI/T5QhQEHiky6g9eiwlBOm2CVdh2zug2vV5q6W90EMUBBhCUm2VtSrCofUnZBXVdBVWLr6hdeVEJWJhB5VcFQ5/JeYgngrMHQO5QO/C1H133QXjh32oaOCceGGfaJZ4jZqxB7IA8CwgAyieKhh+I7K4hBJgRlBqFYi6CDQvrsJ3UDVb+ILNmo8SSeqVtzGo822QbxpHNB5xR1UXtBuWkWWbwVd03JXzsY/UQLgIMaca+m+Gm/wCExBWiw2dh7QH03tcOLbgmJ2KzWd4LVVdsLzc8YmyKyTMy1mmZc0mxEkgb8e2x9igY5lmYLfTq1bCYj3Aj4I7KKzTREm/Pm48fkPiUiz3G6HmGgE2MgcOP1ugcFmLnNhtocT0mIk8fiUGvwVYknSdj72/fyWiymtO6x+TktPOfdaTL3w4ckGroSOyMogzIhD0gCBAgoui08/l/dAdRcQNlJ1TofgvaTLbqflnmgqMnoqnNKL8o81CoxBj88pfanqAfy/JAeWnniGn6mnoR8P8AKVaUAdVqqqG0Iqs1UhiCNCjKrqsujG2Cpc1AFWpKLaSNLV6GIAKtJL8d6QCnb2pbmFIObHIoAGNm6PpNshqZiAi2uQG5XX0PlB5qQahK+Y66hXdIJQVUir2FU0yCr2NQWsRVJBtaofzF4G6A7H0JaRsk9LBaTImAntQajBt3VNXCnYGyCjCMMFxVAoy4unii6IIseyuNKAEGC/iDgdRpNG7yB7kgD6pFTpFldzAbMsCb6m7XPO/zW28YUCBSdxFQe0m3zhYXMKb2ubU3h1xzHEfBAdmRbAfqLjF9QuTwgfFWZPgoAcZE8OnD6/NTpYNtRjXcHCR2mx9wE0oUhAHLggKoVg0BOcszBod6iIWXxcjZAea5B1Sj4jpAwHjvP1CNpeJqYcG62EnqPoFy7JMJXxLi2mWMjcmJ/fstUf4eV3tvVDjHGG/CAg6JQzdvEwjqGNDtiF+fKn8xhqr6WvVoMEarbc22K3v8PM6fVf5dTfh/lB0o4kIOpmLPxCVnvHGIq0y1jA/S5t3NE8YgdYQnh84cs0llZ74vqaRHzQNcxxXmWjbY/kEDoRVeiGuIbZoNuyrNkA1SjK8ZRlEOqtiZB7X+a+ZUEE/kUA9WkqXU0VSBcCYn5ryEAZpKDjKJqlD1DAQU13QEuez0EovEFUCr6SEARoekFWBy9pf1BVSgsa66lUO6G1K57rhAPQB1I7zEK511610oLHVCdkZgsMBfiqqDYRtFyBhXpQ64sqXOEmBZEOxeoSBZUD6oK6oAIHNfAeqyjjTbSBJU8ATs5Av8XZOatB1Rpu0THa9uo3WAfQNWk4H0n8+Y6LsdLEN0lpE8FzLxFl5oVDb7N06SOXI9kC7A1DoaHCHNaG9PSAAR0IU6VYyqqFVuoAX1AiewkfmrSxBsvC5pmzmg9/7rcNy2i5hLabNUfhH6LkmVY8scF07JcxDmggoMhifCFUVC4M1DXPCRfZp5W+q0mT+GS4g1WvAmdJJg9SNRHOy1+EqTwRhAQZ/EeHcKBJo0z3aCl+By2lSqh7Gtb2EfRNM+xQYPok2Fx/rDXNIO9+SDZ4iiHtEqrCYBlOS0XPH5qeFfLOyReNs+OHohtK+IrHy6I5E7vI5NF/ZAB4gxjWPed7277GfdZatm2rdxif8AHshMZrDG0mkvLBBc4jUTxJHMm/ukeEDnEjrF+B2hBp8PiZ5k8HRz/CByTSnV07l3w/NK8LiGgBoF+XXa3w+ScUq5Bhwvy3vOyBphPU2dh8CjHAEXE90to4lpkbHa9rohpIt++KAfF4OAS34foktd8rQ+eNj2SjMsPfU0W4/qgXuHBChlyOSqx9VwcIU8PWjUSgrBhygUPTxEuRMIJ4OhqcArsXhw1/siMmHqurs1pjWSgQVXSURRbAVLRJRIagtYUTTJQ9MIliADwXmprYdpJuBBT/dw6LnP8LccB5jHHqF0HDVQJMoKseHW4RdE0RaeKU5rnlIC9RuobgG/wWd//cjWA2kdIMEl0W5gAGUG11rK+I8aKlQMaZawGSPxHf4IbGZ++tZp0sPBv5n9FVSogXQUVGCNtoI77fRRC+xZiVXSqgoIuN1ovD+alllnSbojBPhwQddynMNQEp1TxErE5RVhoutTgsVZBivH+fupYpjL+WwAuj8RvfpClh/HmFqODZl3AGx9k38UZMzFDU2BVFgeBHAOHLqsUfAFaqYNJvR2oCD0Iug6vlGZ03Mlrguf5vj/ADsVia7Xam0Yo0jNhABqlvd1p6FaTKvD1PA4So958yo2m5xe/hDbBvQRvuuQZLmBaxwOz3Od8SgOfVqaiSSCRP0UsveaMPJsTx6qzC0TWbIIA5lM8DlIxLxq/wBFsANFi4gAEk8GiOFz9Q9yoONQVBak0GJ/rcRFuMCd+iejEVNUNgjjx/Jc7zbOq+pzaFTRTBIaABOkExcgx7ITLfGGLoPkv8wcQ+/wduCg6tV/FPqO4O8+/Be4bGlpg/47hIcgzz+dpuLoY9rt97HaTx+CYYqxaSRPOdxeP31QMa2IBF/ioDFQ2CdtuyVuc5xloPwVOLxkTuOnJARi6cHpuEirY374COOMLmkdEBluGDg4m6BdQxcEd4T4PkSsviaJD46p7ScS0dkDnLcRcL3N8Vcpdh36SELhqmvEPYUF9I2lWsqryvR0mFTCA+m9F0jKXUkbSlAiybwrSw5c9orVHNgcAJ4xcDjxP0K1eGPJpB3giP8AMdJCpMiGj2H6c1bh6/Hf3/YhAn8R0KXmUHVKbSXP0lxHDS4wfccVg/GGS/y1QPZJovu2ZseQPLiFvvGNEPpCLFpDhHPjEoepSGMw5pPFwPSbWI5fvig53l2PO3HgefdabLseKnp2dG3PssfmuXPw1XQ+3EHcEcCOiPuKLKzHXDtJixa4cOoIuD3QPsYEq83SY+CKpYzzGibFD16U2+BQWsqyiaD/AFJM2qR3RuGxIQdGyivqbATn+Xrlh8tzGnm6foFj/DuJWvoteRY2KBKcJmQdapQPX1H5WTzK8LmQIc/EYbTxAY6fm5V18jrVIh+jqgc9yh1Ci59THVNQB0NHpk8BYyUCX+LOJrNa1n8w8te10tENbIiBAuQZ2JKzPh+ia4ePSNIF3GJJ5QCeCQZvjXvqw97nBotqJP1KryvMnUqzXjYH1Dm39eKDouW5caVLyi5hmdVjsTNgeK1WU0GsaCLWgDkGx+iQYHEUqo8yk7WOY+hHBN6ld4DS0NI2LSY+EAoOW+L8M7D1DY6HnW08NMy4H6JZi6rSCBsP7R9R8V0HNcBQx7wxz6jS0QKcta5sGXQYOsT0nqlmZeA9DNDHNY0kEufJPp6CJQeeDcI+nR8y0VCQATuG8h3n4rW4ik3RO2kHvHGIm0pI7Etw+HZT1+lrWtdUd6bCBYTb48UL4nzHRREXL2gMIOw3J62g/BAbWzN4OrzIvEXEn/aOUBDYjEk3dwC5hiq9TXr1EkGQ7jINlqvDme+cCyoftBx4OG09CED7L6+tzwODSf0+qN8P1JaecoTw8GBxLtx0tPGSi8HgnU8Q4Eel10AeYUx5wnimBERCjnuG2cOBVtA6mgnkg9cy0pfkFEnE1H8rJliHeglfeGMMfKdUP9RKA/MKQJBQ7sLZF1zICqBQQoUUdToKmkjaKBZWriAJ47234zvG8+6iK+mW2JvJPXvv79Ev/mSJEA3uYt7nqFbr9XpIEzcnflMboC8aPMpFsHa07/4SXJqrmGCesbgDnE2RNHG+rTyM9entdCZjT9ettoI9QHXe25QB/wARMP5lFlRoEMMnnpcALdJhYrI8cGB7Kgmk6A6N239L28yOXIrp2FqamaXQWkEEEWIO4Mjv81zjH5e2jjvLF6btp/C8EQe35IHVPCaOIIiQ4bOadnDoVNzEB4YxpjyKkuDSdN7tv6oncHl7p1WZxEFpJAcNpHDoeiBNi8PKU1i9lxcLSVm2S6u2UHmU+INBEyF0DIvGTOJC5RWwrpsFT5btUckHeMX44pMZLSHO4N/Xoud514hdWeX1HifoOQCxYrun7xjuo1Tsg6F/DjJsHjq1b+Z1OdA0MDnMEcXamkEnhH62l/FvwrSwhoVcO0MpP+zLBwe0SHSbkuEzJ/pHNYjI8zqYeoKlJxa4bH8lrfFPicZhggHemtRqNeRwc0gsJHWXCyDI4HNatBznUiQHTIO0dY49V1/wHmH81hQ6p/qjU13J0bED4fFclyTLKmJq+WwAO0lxc4wAwRLj8Rbqup5Boo0mBm4BGiwMiNRgbXN+6Bk3KGa2u0tlonaCHmZI+XwWU8f+LTTqeRSAdUbGpzrtbIkADiYM9JC2WNxxpmnsXVHev/aIJO3EALluc5JUdmNUOMNc91XWbgU51DtwbdAuw2YGrU1YuoTTF4tc7wABE2iTYAn3beJ8vfow+Ic1zw+ixri4TDo1XEcQT8FnaGXmvXFBl3uqES0WAJ9TuwuutZ/iqTKQwxa90tAGkTBYJB1bAgDZByE3tCnkmHLX1Kv9FNpk9T90dVpcdRAphrAx2reoYNj7yFfRwbBhn0QAYBJJH3qlod7EADsga5C3WGtaAA4QffePZays28ws54SoEEEiGtHzNgFoq7kAGZ09TCEJhqUMhHvMqtw4IE9bEQxwK0uHZowzBzWcdhpdB5pvUxcaaZHCyCypX4QvmtUvKvKuY1BCky6PpNQulFUig563Gw8sJAkS2frJ43+avZWMWIg26E39Mc91l84r6qTK7fvMcAfoZ9wmWGxGoB7eQPSd7/vggOxji1wdAHIj58r3PwCLwmJ1NF7kOvyIkCR24JdiartPOLk/SB++KCy3FaXm2xgfT9figfYDGFtjNx3jsDwtZZbxs0jG03/0u0lp53g/NPKtSY2425kT+7JX4wpasMx9tdIgwJ+6bHfrFuiBTJZWcWn1B5PeTMLaOoO/lqdTSQ17nHn6rAz7c1hmumu0j+oNJ+EH6LsmRMp1KZw1T7puzgWuAn5hBz2s7gpUqPNWZpRDKzmTIaSJVrIhAHmNRtOmTAnYd0gpNmeatzLF+ZUMfdbYd+JUWbIAKlJU1hZqNruQuId93sgqpOupUn3VTN1403Qa/KMe2k5lQAOc3VaY3aRuO8+y2GSUqFeuajK0CjQIp050vfUcCTINyA4TbcuHK/J/NI4rx9cnuOKDrvh/HVG4XEV8R68NTqBlJ0w4uJDXNaT98a5Fzzg2R+FoUcWBiGNLgWw4uBa4FpMtPKCI5d1yfEeJKz8DTwJA8mnVNQG+rjDTeNILiR36LY/wZy+pWqVqhe8U6IAIaR6nP4ODrOgN4ibiDwQaTL8mpYaTQpNbrJEyS4wb6nOJtPAWQucZi2kAXu+6ZAncnc9Ub48xVWhhn6BLgGNDoj1PLiYBuYAJHdcow2DxD9ZdqOsaS5xmBIJidto7SgeY2q3VraI1Hbh0J6/onGV0vRLuNz7fv5pBgsAWloc4nktrlWDDoLvut/p5mxE9OiBngsPoaBxNz78PYKeIdZXOcg8Q9BXMBVl6+LpUXMQfU6ckI3M6Q9HNCU6kImrW1QeSCOJr6WyiqBloKX45mphCtwOIhgG8IGBVjdt0IKsq6ieqDlj6DQ9zHToqAgwdp4/p2VGRAhrqZMlhc2xseIPWUfmLdTZ4iCY4fry7pbQcRiCdtTWuMXuLGD3AQOC4QZ3GyVV36Hm+8W3t0+CbNZvefn+/7pfjad5ibfvtsgZ4aoC2I+kj9wrsRTD2uY7ZwiText7bn3QGVkfuLjgQPn3TAnjcxeN5HKPyQYzA0neZQH9TXlh/7Tf6FdHw+ILXh4dJEX5RtHMd1ksPR/6pwNjIqD3aRI+Ep1rGrYxw/vG3NAFUY59U/iLihs/xJpN8sWebdhxKJzSoabvMBtEyOiy5c+sTWdJ1ON/yQTw5ABAAvx991IGJVFPcqQcJQRrgkIXG0iHAHkEzwdHW9reZCjnzZrvA4QPgECVu6nhmy5VndX4I+rugIdhkb4ayttet5TpGoEAjg7geqZUcK0i6jhsR5L5pAl4NgOHfkgtxnh1+F1CqwuA2cBYjn0UPDGLr0q2rDOLfU0ubctdpPpDmj7257TZNqNPFY4xVfFMG4GxP5rcZJktKg0BoExc8UCmthK2JqGpiHG8EsFhYQAB297lRzPy6YsBpiLJ9mtYNGqYICW5VlJfU86sCIuxh58HEfqgXZdkTnnU8Op0+A2c7/wCR3v8AVaBtFrQGtEAbD+/FHPCEIlBB5QVciUVVQVZqDyVWXr2FW6yCNWrAXtCqVDy5RFOnCCjH1SGEqWRs+zBPFV5kzUA0cUdhqeloA4ICgVZQbdRpOXuuCgwlRgE9RPLoNuO3VIMUYr0wYNi13GbiPnK0sen3/IrKY8+uieMn8kD1gj7w4QOk9OCGxfDvvtdE4YywE3Mce6qx4t7fkg+wdjeABa3Kb2TXDPaLzN7dklw19U80cNj3P1KCGPOjFUoAuwj2EQJ7uKLxFUAj2IA/f7hAZsftaH/f/wCivxY9Q7IA/Ezz5DiNp+qB8PAHDO39LnXHUBFZ2P8ApHH/AI/UITwn/oVu7UC5x/NRD1Hie/5rxm4QO/D3+sOjSZS3FOmo87y531THw999x/2lKmn1H98UC6sfUVZgrvAVdfc91blv3kDwViXBgO+61eBy8Q1jGwTEnjH99ysdgfvjv+a6ZkAugbYDBCmzS0bIoVAxpPuoNN3dkPXu9oNwXNBB4jUJlBdk+XOqPNerOmfs2HYgbPI+g904q0gjqiFrIAajUO9tkVVVR2QLzTVFRqNqIVyAVwhU+WSr3KZCCkMXpVhUUH1KlJlFNCgwKQQfAKNQqNQ3QuIKD//Z)

**Pull request merged**